### ISSUE TITLE
[Feature Request] iCloud (or other) settings/waypoints/tracks sync

### ISSUE THREAD

**Issue by juliusvaart** [osmandapp/OsmAnd-iOS#2077]
Would be great if settings, waypoints, routes, tracks, favourites, maps could be synced with iCloud or Dropbox or…

A possibility to include/exclude maps/favourites/etc. would be nice.

This would make OsmAnd really awesome! Even more than it already is!

**Comment by Nikita7x** [comment]
Oh, for real. Just ability to use iCloud folder to store contents of OsmAnd will be perfect. Because Favorites in OsmAnd are currently stored inside "OSMAnd" folder under favorites.gpx

**Comment by Nikita7x** [comment]
> manually

Yeah, it seems like a "solution" but still, it is Manual. It's really hard keeping a zoo of devices with up-to-date favourites.

**Comment by juliusvaart** [comment]
Saw in the testflight version that OsmAnd Cloud is coming. That would solve this!

**Comment by drtbyk** [comment]
OsmAnd Cloud service does not solve this request in my view. It certainly is an option but I would prefer to keep my data local to my devices and not use third-party cloud storage since I already use iCloud Sync; which is encrypted.

**Comment by vshcherb** [comment]
You can keep local with manual exports. Also OsmAnd Tracks folder is accessible and could be synced with iCloud

**Comment by drtbyk** [comment]
There seems to be an issue when doing the file sharing discussed above. When I copy the OsmAnd/GPX/my_route.gpx file from my iPadPro over to my iPhone 14Pro Max, OsmAnd edits my route path as you can see in the screenshots below. Why is OsmAnd editing my route?

![IMG_1183](https://user-images.githubusercontent.com/8745955/221742939-2e68b52f-a3a1-4e90-a07d-54b2f5d1f72d.jpeg)

**Comment by vshcherb** [comment]
Yes, it doesn't copy automatic and not possible to create automatic symlink.

**Comment by drtbyk** [comment]
OsmAnd Cloud Sync is also not going to address syncing the users data between user devices since the user has to resolve all data conflicts - making this a one-device "cloud-backup" service.

**Comment by juliusvaart** [comment]
An improvement would be automatic sync with OsmAnd cloud that always uses the newest changes.
